src/auth/custom/custom.authorizer.ts

```diff
@@ -51,7 +52,10 @@ export class CustomAuthorizer implements IAuthorizer {
     public generateUserSessionToken = async (user: CurrentUser): Promise<string> => {
         return new Promise((resolve, reject) => {
             try {
-                const token = jwt.sign(user, process.env.USER_ACCESS_TOKEN_SECRET, { expiresIn: '90d' });
+                const defaultSessionDuration = ConfigurationManager.DefaultSessionDurationDays();
```

Comment:
Send the current environment to this function and get session duration. That way we do not have to worry about having separate file content for each environment.